batch/batch/server/deserialize.py

```diff
@@ -0,0 +1,91 @@
+from datetime import date, datetime
+import re
+from kubernetes.client import models
+
+
+def deserialize(api_client, data, klass):  # pylint: disable=R0911
+    if data is None:
+        return None
+
+    if isinstance(klass, str):
+        if klass.startswith('list['):
+            sub_kls = re.match(r'list\[(.*)\]', klass).group(1)
```

Comment:
Similarly the kube library doesn't use raw strings. I don't think this will introduce a problem, but wanted to mention it.